src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -23,8 +23,7 @@
    title="<%= Const.Tooltips.FEEDBACK_SESSION_DELETE %>"
    data-toggle="tooltip"
    data-placement="top"
-   <c:if test="${actions.allowedToDelete}">onclick="return toggleDeleteFeedbackSessionConfirmation(${actions.toggleDeleteFeedbackSessionParams});"</c:if>
-   <c:if test="${not actions.allowedToDelete}">onclick="return false"</c:if>
+   onclick="return toggleDeleteFeedbackSessionConfirmation(${actions.toggleDeleteFeedbackSessionParams});"
```

Comment:
Do you know how changes like this occur? This is definitely not a part of this issue.